library/hawkular_alerts_group.py

```diff
@@ -89,13 +89,38 @@
 
 
 class HawkularAlertsGroupTrigger(object):
-    """ Hawkular Alerts object to create group triggers in Hawkular
+    """ Hawkular Alerts object to create and delete group triggers in Hawkular
     """
     def __init__(self, module, tenant, hostname, port, scheme, token, context):
         self.module  = module
         self.client  = hawkular.alerts.HawkularAlertsClient(tenant, host=hostname, port=port, scheme=scheme, token=token, context=context)
         self.changed = False
 
+    def delete_group_trigger(self, group_id):
+        """ Deleted a group trigger in Hawkular Alerting component
+
+            Returns:
+                whether or not a change took place and a short message
+                describing the operation executed
+        """
+        try:
+            self.client.get_trigger(group_id)
+            try:
+                self.client.delete_group_trigger(group_id)
+                self.changed = True
+                return dict(
+                    msg="Successfully deleted group trigger {group_id}".format(group_id=group_id),
+                    changed=self.changed)
+            except Exception as e:
```

Comment:
> Why do you need two try block?

I'll drop one of the `try`s, thanks.

> Also,  why do you raise for HTTPError with code different than 404 but for a general error you call self.module.fail_json

If the group trigger doesn't exist (`HTTPError` 404) it isn't a fail in the ansible run, actually this is the desired state (even if no change happen).
If the delete action failed we want the module run to fail.